katalog/harbor/core/deploy.yml

```diff
@@ -90,5 +95,8 @@ spec:
         - name: token-service-private-key
           secret:
             secretName: core-root-ca
+        - name: ca-download
+          secret:
+            secretName: core-root-ca
```

Comment:
where does this secret get created?

Reply:
This secret is created in the _katalog/harbor/core/pki.ym_l file

Reply:
ohh right, I was skimming for a secret but the secret itself is created by cert-manager 👍